### ISSUE TITLE
conversion code for batch norm layer is missing in conversion.py

### ISSUE THREAD

**Issue by mominali12** [tinysystems/FreeML#3]
Please add the conversion code for converting  batchnorm layers
